docs/FAQ/FAQ_How_do_I_use_Edge-IE_as_the_Browser's_underlying_renderer.md

```diff
@@ -0,0 +1,41 @@
+# FAQ How do I use Edge/IE as the Browser's underlying renderer?
+
+Since Eclipse/SWT 4.35 (2025-03) Edge is used as the default browser in SWT. For older releases, browser instances using Edge could be created with the style `SWT.EDGE` (since 4.19) or setting the Java property `org.eclipse.swt.browser.DefaultType=edge`. 
+
+## WebView2 Component Provision
+
+Edge rendering backend uses the WebView2 component, which is based on but distinct from the Edge browser itself. SWT relies on a WebView2 component being available on the current system.
+There are different [ways of distribution for the WebView2 component](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution). On most systems, the component is already available due to the `Evergreen runtime distribution mode` ensuring the availability of an automatically updated version of the library on Windows.
+
+If this is not the case or if other versions of the component shall be used, these options exist:
+* A stand-alone runtime installer, either web or offline ([Download the WebView2 Runtime](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#webview-title) from Microsoft).
+  This runtime will be shared between all applications on the machine and will auto-update itself independent of your application.
+* A fixed-version archive with all the necessary files (same link as above).
+  This is a complete, fixed set of files to be included with your application. Unlike the first option, you have complete freedom in bundling, packaging and updating it.
+* Beta, Dev, or Canary version of the Edge browser (<https://www.microsoftedgeinsider.com/en-us/download>).
+  This option is convenient for testing, but production deployments should use the previous two options.
+
+SWT will automatically locate installed browsers and runtimes (using the `WebView2Loader.dll` provided via the SWT fragment for Windows).
+In case you want to use fixed-version binaries or override the automatically chosen version, set the `org.eclipse.swt.browser.EdgeDir` Java property to the directory containing `msedgewebview2.exe`. For example:
+
+```sh
+java "-Dorg.eclipse.swt.browser.EdgeDir=C:\Program Files (x86)\Microsoft\Edge Beta\Application\88.0.705.29" ...
```

Comment:
Starting Eclipse via a `java` command isn't that common.
Should the example just mention the 'system property' and not give a command line example?
Or, alternatively, mention to modify the eclipse.ini`

Reply:
Makes sense. This is probably only relevant when having a pure SWT application. I have reduced this to a sentence mentioning the system property and referring to the `eclipse.ini` for Eclipse products.